cmd/ova-serial-api/main.go

```diff
@@ -45,9 +69,42 @@ func main() {
 
 	for _, slice := range slices {
 		fmt.Println("Slice")
-		utils.Print1dSlice(slice)
-		var filteredSlice = utils.FilterSlice(slice)
+		utils.Print1dIntSlice(slice)
+		var filteredSlice = utils.FilterIntSlice(slice)
```

Comment:
В Go считается хорошим тоном по возможности делать так:
```suggestion
		filteredSlice := utils.FilterIntSlice(slice)
```